gateway-export-plugin/src/main/java/com/ca/apim/gateway/cagatewayexport/tasks/explode/linker/ClusterPropertyLinker.java

```diff
@@ -22,10 +22,8 @@ public void link(Bundle filteredBundle, Bundle bundle) {
         Map<String, ClusterProperty> clusterPropertyMap = filteredBundle.getEntities(ClusterProperty.class);
         Set<String> idsToRemove = new HashSet<>();
         for (ClusterProperty clusterProperty : clusterPropertyMap.values()) {
-            if (clusterProperty.getName().startsWith("ENV.")) {
-                filteredBundle.addEntity(new EnvironmentProperty(clusterProperty.getName().substring(4), clusterProperty.getValue(), EnvironmentProperty.Type.GLOBAL));
-                idsToRemove.add(clusterProperty.getId());
-            }
+            filteredBundle.addEntity(new EnvironmentProperty(clusterProperty.getName(), clusterProperty.getValue(), EnvironmentProperty.Type.GLOBAL));
```

Comment:
During export, all cluster properties will be treated as environment properties, does this mean static.properties will no longer be exported?